style.css

```diff
@@ -0,0 +1,186 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100;400&display=swap');
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-wrapper {
+  max-width: 400px;
+  height: 92px;
+  background: #3c3a39;
+  display: flex;
+}
+
+.navtop {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 10px;
+  color: #fff;
+  position: absolute;
+  width: 400px;
+  height: 44px;
+}
+
+.time {
+  height: 16px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  letter-spacing: -0.28px;
+}
+
+.navbottom {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #fff;
+  position: absolute;
+  width: 363px;
+  top: 44px;
+}
+
+.logo {
+  position: static;
+  height: 24px;
+  left: 12px;
+  top: 10px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.content-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  position: absolute;
+  height: 590px;
+  width: 400px;
+  top: 92px;
+  background: #1c1a19;
+}
+
+h1 {
+  position: static;
+  width: 327px;
+  height: 124px;
+  left: 24px;
+  top: 120px;
+  font-family: 'Crete Round', serif;
+  color: #ff6b00;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px;
+}
```

Comment:
- [ ] It will be nice to fix the margin of the header so the design looks like the one provided in the Figma.
- [ ] The same thing for the social media icons 

<img width="275" alt="image" src="https://user-images.githubusercontent.com/77190203/148378118-74a28c13-7ba6-4e5f-a3d1-8cc13f654a3e.png">